coreSDK/JavaScriptSDK.Interfaces/ITelemetryPlugin.ts

```diff
@@ -1,17 +1,16 @@
 /// <reference path="./ITelemetryItem.ts" />
 /// <reference path="./IConfiguration.ts" />
```

Comment:
do you still need these?

Reply:
I'll clean these up in next pr (coming up for webpack changes)